ndcube/ndcube.py

```diff
@@ -963,7 +963,7 @@ def _new_instance(self, **kwargs):
     def __neg__(self):
         return self._new_instance(data=-self.data)
 
-    def add(self, value, handle_mask=np.logical_and):
+    def _arithmetic(self, value, op, handle_mask=np.logical_and):
```

Comment:
The args need to be in the order that you call them with below.  Also, do keep in-line with methods starting with verbs, let's rename this method.  So:
```suggestion
    def _operate_arithmetic(self, operation, value, handle_mask=np.logical_and):
```